### ISSUE TITLE
Bug - PHP Warning: “continue” targeting switch is equivalent to “break”. Did you mean to use “continue 2”

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#2743]
Reported here: https://wordpress.org/support/topic/keep-getting-this-error-2/

[24-Jul-2019 17:38:59 UTC] PHP Warning: “continue” targeting switch is equivalent to “break”. Did you mean to use “continue 2”? in /homepages/43/d695121066/htdocs/Body/wp-content/plugins/all-in-one-seo-pack/modules/aioseop_robots.php on line 510
[24-Jul-2019 17:38:59 UTC] PHP Warning: “continue” targeting switch is equivalent to “break”. Did you mean to use “continue 2”? in /homepages/43/d695121066/htdocs/Body/wp-content/plugins/all-in-one-seo-pack/modules/aioseop_robots.php on line 530
[24-Jul-2019 17:38:59 UTC] PHP Warning: “continue” targeting switch is equivalent to “break”. Did you mean to use “continue 2”? in /homepages/43/d695121066/htdocs/Body/wp-content/plugins/all-in-one-seo-pack/modules/aioseop_robots.php on line 536
[24-Jul-2019 17:39:02 UTC] PHP Warning: “continue” targeting switch is equivalent to “break”. Did you mean to use “continue 2”? in /homepages/43/d695121066/htdocs/Body/wp-content/plugins/all-in-one-seo-pack/modules/aioseop_robots.php on line 507

**Comment by michaeltorbert** [comment]
This may be similar to the issue we had in #2129 for a different file.

**Comment by arnaudbroes** [comment]
@michaeltorbert I couldn't find any `continue` statements in switch structures either so I wondered whether the user was using an older version of the plugin (since we didn't ask) and then I remembered we refactored the robots.txt module in v2.7. I had a look at v2.6.1 and saw that version did have `continue` statements in switch structures in modules/aioseop_robots.php so I'm guessing he was using an earlier version of the plugin. In any case, we do not have `continue` statements in switch structures in aioseop_robots.php in our current version so this issue is invalid.
